src/App.js

```diff
@@ -1,13 +1,28 @@
 import React from "react";
-
 import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristicData from "./destinationInfo.json";
+import TouristInfoCards from "./TouristInfoCards";
+import Restaurant from "./Restaurant";
+import Footer from "./Footer";
 
 const App = () => {
+  const countryInfo = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com ",
+    "0123 456789 "
+  ];
+
+  const touristicInformation = TouristicData;
```

Comment:
We already have a variable `TouristicData`, so do you think another one is necessary?

Reply:
DEfinitely not, why I did this :(.